src/main/java/frc/robot/Robot.java

```diff
@@ -78,7 +78,7 @@
      */
     @Override
     public void robotInit() {
-        Logger.configureLoggingAndConfig(this, true);
+        Logger.configureLoggingAndConfig(this, false);
```

Comment:
Don't we want separate tabs for each subsystem?

Reply:
Setting this true creates 2 tabs per subsystem, which is not ideal.